torch/_inductor/codegen/common.py

```diff
@@ -1313,9 +1316,13 @@ def cpp_argdefs(self):
             call_args.append(self.wrap_ptr_arg(outer, dtype))
             arg_types.append(f"{cpp_dtype}*")
         for outer, inner in self.sizevars.items():
-            arg_defs.append(f"const {INDEX_TYPE} {inner}")
+            if symbol_is_type(outer, (SymT.UNBACKED_FLOAT, SymT.FLOAT)):
```

Comment:
Do we need to support other types?

Reply:
let's parameterize tests and see if breaks